python-backend/app/api/documents/mines/models/mine_document.py

```diff
@@ -36,3 +36,8 @@ def find_by_mine_guid(cls, mine_guid):
     def find_by_mine_document_guid(cls, mine_document_guid):
         return cls.query.filter_by(mine_document_guid=mine_document_guid).filter_by(
             active_ind=True).first()
+
+    @classmethod
+    def find_by_document_manager_guid(cls, document_manager_guid):
+        return cls.query.filter(str(document_manager_guid) == str(document_manager_guid)).filter_by(
+            active_ind=True).first()
```

Comment:
This doesn't appear to be used, do we have a use case for this method? Also is this looks like it doesn't work as intended. if it's needed it should look like
`return cls.query.filter_by(document_manager_guid=document_manager_guid).filter_by(active_ind=True).first().`

Reply:
I actually thought I had already removed this. I'll take another look at it

Reply:
yea, removing it seems right to me, thanks.